devicetypes/smartthings/zigbee-siren-src/zigbee-siren.groovy

```diff
@@ -0,0 +1,150 @@
+/**
+ *  Zigbee Siren
+ *
+ *  Copyright 2018 Samsung SRBR
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+	definition(name: "ZigBee Siren", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "x.com.st.d.siren") {
```

Comment:
You'll need to rename this to "Ozom Smart Siren", change the directory to `ozom-smart-siren.src` and the filename to `ozom-smart-siren.groovy`. Also please add `mnmn: "SmartThings", vid: "generic-siren-2"` to the definition to associate ui-metadata for the new app.